app/views/making_a_differences/_form.html.erb

```diff
@@ -0,0 +1,28 @@
+<%= form_for(@making_a_difference) do |f| %>
+  <% if @making_a_difference.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(@making_a_difference.errors.count, "error") %> prohibited this making_a_difference from being saved: %></h2>
+
+      <ul>
+      <% @making_a_difference.errors.full_messages.each do |message| %>
+        <li><%= message %></li>
+      <% end %>
+      </ul>
+    </div>
+  <% end %>
+  
+  <%= render partial: 'helper_fields/text', locals: {form: f, field: :comment} %>
```

Comment:
these render partials should be wrapped in row/column divs so that you can explicitly specify their layout.

Reply:
row and column are added